src/main/java/com/google/sps/data/ComputeAPIClientInterface.java

```diff
@@ -11,31 +11,37 @@
    * are executed.
    * @param {String} operation - the name of the method that should be called.
    * @param {String} instanceName - the name of the instance being operated on.
-   * @param {Optional<String>} password - the password needed to access this 
-   *    instance. Not all operations requires a password.
+   * @param {Optional<String>} sessionId - the session id associated with
+   *     this instance. 
    */
-  void helper(String operation, String instanceName, 
-    Optional<String> password);
+  void computeManager(String operation, String instanceName, 
+    Optional<String> sessionId);
 
   /** 
    * Creates a new Instance.
-   * @param {String} password - the password needed to access this instance. 
    * @param {String} instanceName - the name of the new Instance.
+   * @param {String} sessionId - the session associated with this instance.
+   * @return Operation object that can be used to wait for operation completion
+   *    and to check whether operation failed or succeeded
    */
-  Operation createInstance(String instanceName, String password) 
+  Operation createInstance(String instanceName, String sessionId) 
```

Comment:
All these functions that return Operation should not. They should be void, for now. In these functions, just wait until the operation is complete